### DISCUSSION TITLE
[Question][Hw4] Which operations are valid for Capability?

### DISCUSSION THREAD

**Discussion by m-spitfire** [prosyslab-classroom/cs348-information-security#711] [Q&A]
Name: Murad Bashirov

Hello. In the documentation it's written that we need to implement 7 in the first sentence, 6 commands are written for describing the type command, and there are only 5 in the table. So which commands are exactly needed for capability? Thanks.
![Screenshot from 2023-05-10 21-11-09](https://github.com/prosyslab-classroom/cs348-information-security/assets/66242799/092b6d6f-41fe-40a6-8aa8-921d2b2d5ae4)

**Comment by sujin0529** [comment]
Hi,

You can implement 6 commands.
`Create, Confer, Remove, Open, Read_with_capability, Execute_with_capability`
Sorry for confusion.

Thanks.

**Comment by m-spitfire** [comment]
Thanks.
